.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: rust
+rust:
+- nightly-2016-04-11
+before_script:
+- |
+  pip install 'travis-cargo<0.2' --user &&
```

Comment:
@killercup I glanced at travis-cargo's readme, but I'm still unsure what benefit it provides over running cargo directly in this case. Can you explain?